src/php/ext/grpc/channel.c

```diff
@@ -157,6 +162,23 @@ void generate_sha1_str(char *sha1str, char *str, php_grpc_int len) {
   make_sha1_digest(sha1str, digest);
 }
 
+// Return the string format of connectivity status. Only used for testing.
```

Comment:
Add a TODO marker? Why do we need this?

Reply:
We don't need to actually. It's just easy to be read during the PersistentListTest.php.
But it's the same as Grpc\IDLE, Grpc\CONNECTING.
I'll remove it.